target/consensource-ui
# Perichsen/linting rules

## Proposed change/fix

Update linting rules and apply fixes. 

## Types of changes

What types of changes does this pull request introduce to ConsenSource? _Put an `x` in the boxes that apply_


## How to run/test

- `npm run lint` and verify there are no errors
- `npm run test` and verify all tests pass
